en/flight_modes/README.md

```diff
@@ -249,6 +249,17 @@ th {
 </td>
 </tr>
 
+<tr id="position_slow_mc">

```

Comment:
FYI, updated the other pages that mention flight modes to cross link. Note that I'll be reworking all these pages in near future when MC, VTOL, FW all get their own pages, so this is to make sure I don't forget to add later.